interfaces/cython/cantera/reaction.pyx

```diff
@@ -795,6 +795,44 @@ cdef class Reaction:
             if products:
                 self.products = products
 
+    def __init__(self, reactants=None, products=None, *, equation=None, rate=None,
+                 Kinetics kinetics=None, init=True, legacy=False, **kwargs):
```

Comment:
Would it make sense to write this as
```Python
    def __init__(self, reactants=None, products=None, rate=None, *, equation=None, 
                 Kinetics kinetics=None, init=True, legacy=False, **kwargs):
```
Also see comment in `blowers_masel.py`.

Reply:
I've adopted this as the signature for all non-deprecated `Reaction` types, as part of making the `rate` argument required.